ios/Podfile.lock

```diff
@@ -1399,8 +1399,8 @@ SPEC CHECKSUMS:
   CocoaAsyncSocket: 065fd1e645c7abab64f7a6a2007a48038fdc6a99
   disklet: e7ed3e673ccad9d175a1675f9f3589ffbf69a5fd
   DoubleConversion: fea03f2699887d960129cc54bba7e52542b6f953
-  edge-core-js: 8ffc2832bff1e8ee96dd1af557f7dad90e2d1895
-  edge-login-ui-rn: c22ef0606f18c9427513a325fcf0290ccdceaccf
+  edge-core-js: b7c6fca8bbaf9848a5b4e1871f8ae7b95b5c8f6d
+  edge-login-ui-rn: eaeb2fde5abcdcac0bc5d08e9c14554d9814479f
```

Comment:
Separate commit for the lib upgrades?

Reply:
Not sure how / when those got in. I should probably commit those to straight to master, assuming we somehow have an out-of-date podfile.